web/src/services/api/propertyApi/processNewProperty.ts

```
@@ -0,0 +1,138 @@
+  if (!schema) return;
+
+  const allItems: Record<
+    string,
+    {
+      schema: PropertySchemaGroupFragmentFragment;
+      orig?: PropertyItemFragmentFragment;
+      parent?: PropertyItemFragmentFragment;
+    }
+  > = schema.groups.reduce(
+    (a, b) => ({
+      ...a,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)